### PR TITLE
Creating a pull request for code reviewing

### PR DESCRIPTION
**Pull request by menyagah**
## YouTube Clone:
In this section we:
- Created the html elements
- Styled the page
- Used stickler to validate our code
- Changed the fonts to Roboto
- Worked our readme file and added a screenshot on it
- Hosted our page on github pages
- Created a new branch called gh-pages
- Merged our feature branch to the gh-pages branch.
-Cleared the white space errors that had erupted from the markup. 

## Live Demo

[Live Demo Link](https://menyagah.github.io/youTube_clone/)

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review by Oluwadamilare Tolulope Olusakin(TSE)
Great work @menyagah and @Morrism1. I am currently writing your review and will be done within the hour, please check on it in a few minutes :+1:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
```
Comment (nikasulo) [inline index.html]: #### [OPTIONAL]
Consider loading up your css file before the font files so your page can look great immediately the file is served. Font files can take a bit of time to load up especially with slower internet connections and mobile devices! :+1:

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice work team! @menyagah and @Morrism1 
This is really fantastic all together! Great job :handshake:. Just one tiny issue to address and you can wrap up here :smiley:, please see my comments below.

#### Descriptive PR
You need a more descriptive PR as it is a hard requirement. Consider describing the work you have completed in the branch in as much detail as you can and even include a link to the live project to speed up your review!
screenshot

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by menyagah** [comment]
Thank you so much for the review! Much appreciated! One question though,
you have pointed out on the Pull Requests issue. Kindly if I may ask do you
recommend that we go back and make them more descriptive or is this a
recommendation for our future projects? Much appreciated for the
quick response and outstanding delivery.

**Review by mosesogwo — APPROVED** [review]
**Hi team, Good work :+1:. Your project is approved.**

In future projects, also use descriptive titles for your pull request. For example, this current pull request title could be "Youtube Clone - Complete feature".

You may now merge to master.

**Cheers :smile:**

**Status: Approved!**

**Pull request merged**